Dockerfile

```diff
@@ -3,6 +3,7 @@ FROM quay.io/assemblyline/nodejs:0.12.10
 RUN apt-get update -q\
```

Comment:
This version of node is not supported any more, it doesn't even get critical bug fixes. Do not use this in production. Upgrade from v0 to v6 as a priority if you are.